src/Build/BackEnd/Components/Scheduler/Scheduler.cs

```diff
@@ -1288,20 +1288,12 @@ private bool AtSchedulingLimit()
             }
 
             int limit = 0;
-            switch (_componentHost.BuildParameters.MaxNodeCount)
+            limit = _componentHost.BuildParameters.MaxNodeCount switch
```

Comment:
Can you also say `int limit = _componentHost...` and avoid the definition above?